### ISSUE TITLE
List issues/PRs that will be closed when merging a PR

### ISSUE THREAD

**Issue by fregante** [refined-github/refined-github#7894]
### Description

Initially suggested in https://github.com/refined-github/refined-github/issues/2579

In short, when I click "Merge pull request" I want to be reminded of which issues/PRs will close.

This list can be retrieved via API ("linked issues"), or in the sidebar+first post

### Example URLs

https://github.com/refined-github/sandbox/pull/7

**Comment by SunsetTechuila** [comment]
do you want something like this?

<img width="535" height="509" alt="Image" src="https://github.com/user-attachments/assets/305142b3-4952-478d-ad99-50b31741867e" />

**Comment by fregante** [comment]
Yes that would be good. No "issues" because it can also close PRs; make sure to include the hovercard attributes so that one can see what those issues are. 

However I'm still prioritizing bugfixes because we just have too many broken features to be adding new ones, especially non-trivial ones like this one probably is.
